README.md

```diff
@@ -28,48 +28,125 @@ The function `initializePowerDmsFilePicker` accepts a configuration object. One
 
 `initializePowerDmsFilePicker` accepts a configuration object as a single parameter. This object has the following structure:
 
+### JavaScript
+
 ```javascript
 {
-  apiKey,         // string,   required
-  onFileSelected, // function, required
-  width,          // number,   optional
-  height          // number,   optional
+  // String, required
+  // The API key given to your organization from PowerDMS.
+  apiKey,
+
+  // Function, required
+  // A callback that is called when a user makes a selection.
+  onSelection,
+
+  // Number, optional
+  // The desired width of the file picker, in pixels.
+  // Restricted from 566 to 1051.
+  width,
+
+  // Number, optional
+  // The desired height of the file picker, in pixels.
+  // Restricted from 350 o 650.
+  height
 }
 ```
 
-If you are familiar with TypeScript, the properties have these types:
+### TypeScript
 
 ```typescript
-{
-  apiKey: string,
-  onFileSelected: (fileRoute: string) => void,
-  width?: number,
-  height?: number
+type FilePickerConfig = {
+  // The API key given to your organization from PowerDMS.
+  apiKey: string;
+
+  // A callback that is called when a user makes a selection.
+  onSelection: (response: SelectionResponse) => void;
+
+  // The desired width of the file picker, in pixels.
+  // Restricted from 566 to 1051.
+  width?: number;
+
+  // The desired height of the file picker, in pixels.
+  // Restricted from 350 o 650.
+  height?: number;
 }
 ```
 
-### `apiKey`
+## Response data

```

Comment:
💭 Might be missing a '#' here

Reply:
Nah, the `Response data` section is supposed to be at level 2